### DISCUSSION TITLE
Password Generator UI

### DISCUSSION THREAD

**Discussion by Asterisk-z** [reactplay/react-play#214] [Ideas]
Here is the UI idea i want to implement on the password generator codebase
![passwordui](https://user-images.githubusercontent.com/56372709/169662616-1e07c6fc-97ad-4e7a-b288-42921aba7a31.png)

**Comment by Sachin-chaurasiya** [comment]
Hey @Asterisk-z , this looks great.

**Reply by Asterisk-z** [reply]
thanks @Sachin-chaurasiya

**Comment by atapas** [comment]
> Here is the UI idea i want to implement on the password generator codebase ![passwordui](https://user-images.githubusercontent.com/56372709/169662616-1e07c6fc-97ad-4e7a-b288-42921aba7a31.png)

Looks better. Here is the current implementation. Where is the Submit button in the new design:? Also it misses the special character optin.
![image](https://user-images.githubusercontent.com/3633137/169679680-c16a6135-aea7-4efc-bb2d-a218b820d27a.png)

**Comment by Asterisk-z** [comment]
![passwordui (1)](https://user-images.githubusercontent.com/56372709/169681732-85697292-6e8d-46cd-9cc0-fc259ec49b84.png)

**Reply by atapas** [reply]
looks good to me.

**Reply by atapas** [reply]
Please create an issue and get started on it. Thanks a lot.
